scripts/badger/topups/quarter_2.py

```diff
@@ -0,0 +1,40 @@
+from decimal import Decimal
+
+from brownie import interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+N_WEEKS  = 6
+
+
+def main():
+    trops = GreatApeSafe(registry.eth.badger_wallets.treasury_ops_multisig)
+    dripper = GreatApeSafe(registry.eth.drippers.rembadger_2022_q2)
```

Comment:
should not this be instead for `"tree_2022_q2": "0xfA55d407F48849aE98a6e4f11Dc5C18E035F46Ec"` ?

Reply:
ah yea you are right, it isnt merged yet though, i think that is what confused me